cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less

```diff
@@ -574,7 +574,7 @@
             color: @gold;
         }
 
-        .m-notification_icon {
+        .cf-icon-svg {
             top: 0;
```

Comment:
Are you sure this was needed?

Reply:
Tested this now that it's live (wouldn't work locally) and it turns out this is necessary and may be necessary for all notifications to ensure if there's a heading _and_ content the icon doesn't shift to mid-height of that whole content block. Will add it to the big icons PR.